src/WingetCreateCore/Common/Exceptions.cs

```diff
@@ -0,0 +1,82 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license.
+
+namespace Microsoft.WingetCreateCore.Common
+{
+    using System;
+    using System.Collections.Generic;
+    using Microsoft.WingetCreateCore.Models.Installer;
+
+    /// <summary>
+    /// Exception class for handling installer matching errors.
+    /// </summary>
+    public class Exceptions
```

Comment:
Weird to have this nested like this. Instead why not just extend the namespace to be Microsoft.WingetCreateCore.Common.Exception instead?

Reply:
Extended namespace and created an Exceptions folder containing the exception classes.